_change/in-canada.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: change/in-canada
```

Comment:
If we choose to use the same layout for each of these pages, this part doesn't have to be this way. I just copied it over from the other version of the project like this.